webapp/.angular-cli.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Let's keep spacing the same as in `https://github.com/beeman/fireloop-demo-app/blob/master/webapp/.editorconfig`